bftclient/src/bft_client.cpp

```diff
@@ -57,6 +58,13 @@ Client::Client(std::unique_ptr<bft::communication::ICommunication> comm, const C
   }
   communication_->setReceiver(config_.id.val, &receiver_);
   communication_->start();
+  if (config_.replicas_master_key_folder_path.has_value()) {
+    bft::communication::StateControl::instance().setGetPeerPubKeyMethod([&](uint32_t rid) {
+      concord::secretsmanager::SecretsManagerPlain psm_;
+      std::string key_path = config_.replicas_master_key_folder_path.value() + "/" + std::to_string(rid) + "/pub_key";
```

Comment:
Do we exchange replica main public key if a replica is down in the initial start.
Do we need to add replica main key exchange here?

Reply:
right, thanks